### ISSUE TITLE
Student Gradebook Unstarted Assignments

### ISSUE THREAD

**Issue by eecarrier** [PrairieLearn/PrairieLearn#2166]
It's somewhat confusing that the student gradebook doesn't show anything for assignments that haven't been started.  It'd be nice to still show it, but just show "not started" or something along those lines.

**Comment by firasm** [comment]
I'll add some screenshots to this issue just to make the issue clearer:

## Student View of Assessments page

<img width="1278" alt="Screenshot 2023-05-15 at 4 37 58 PM" src="https://github.com/PrairieLearn/PrairieLearn/assets/2507459/8e1204fe-87fd-48ad-a0bf-083d0ca72053">

## Student view of Gradebook (before starting)

This is when no assignment has been started

<img width="1278" alt="Screenshot 2023-05-15 at 4 38 04 PM" src="https://github.com/PrairieLearn/PrairieLearn/assets/2507459/cbf9a556-da1a-4fd2-9b10-7092e6a19c98">

## Student view of Gradebook (after starting an assignment)

<img width="1278" alt="Screenshot 2023-05-15 at 4 38 21 PM" src="https://github.com/PrairieLearn/PrairieLearn/assets/2507459/351354a7-d58b-43cc-8e65-f1b0d6f2dc9c">

**Comment by firasm** [comment]
And now I'll add a personal opinion: I'm not sure what the purpose of the (current) Student Gradebook is given that all the information from the Gradebook is currently available on the Assessments page.

(Of course, I'd be super interested if the Gradebook was to be revamped to include other things I'd be super interested in finding a way to add Status messages about Feedback summaries, Manual Grading statuses, time spent, number of attempts etc... and which questions have new messages from the Course Staff).

**Comment by czhou578** [comment]
I agree. This needs to be clarified. @nwalters512, do you have the same opinion on this?

**Comment by mfsilva22** [comment]
One main difference from the Assessment page to the Gradebook page: if an assessment is no longer available (for example, an exam that is no longer available to students), they won't be able to see that in the assessment page, but they can see their grade in the Gradebook.

**Comment by jonatanschroeder** [comment]
Another difference, if I recall correctly, is that assessments with multiple instances show only once in the gradebook.
